build-support/bin/native/rust_toolchain.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Nit: This file is missing a bunch of quotes around variables

Reply:
Fixed.